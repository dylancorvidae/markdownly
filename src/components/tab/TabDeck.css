.TabDeck {
  list-style-type: none;
  clear: both;
  display: flex;
  width: 100%;
  border-bottom: 1px solid #428bca;
  padding: 0 10px;
  position: relative;
  background-color: gray;
}