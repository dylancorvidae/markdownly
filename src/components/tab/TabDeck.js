import React from 'react';
import PropTypes from 'prop-types';
import Tab from './Tab';

const TabDeck = ({ titles, handleDelete }) => {
  const tabItems = titles.map(title => {
    return (
      <Tab handleDelete={handleDelete} key={title} title={title} />
    );
  });

  return (
    <ul>
      {tabItems}
    </ul>
  );
};

TabDeck.propTypes = {
  titles: PropTypes.array.isRequired,
  handleDelete: PropTypes.func.isRequired
};

export default TabDeck;
