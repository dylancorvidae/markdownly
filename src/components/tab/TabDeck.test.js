import React from 'react';
import { shallow } from 'enzyme';
import TabDeck from './TabDeck';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<TabDeck titles={[]} handleDelete={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
