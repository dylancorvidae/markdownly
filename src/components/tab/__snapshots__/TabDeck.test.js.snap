// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabDeck component renders TabDeck 1`] = `
<ul
  className="TabDeck"
/>
`;
